extensions/amp-auto-ads/0.1/wunderkind-network-config.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '#service';
+import {buildUrl} from '#ads/google/a4a/shared/url-builder';
+import {dict} from '#core/types/object';
+
+/**
+ * @implements {./ad-network-config.AdNetworkConfigDef}
+ */
+export class WunderkindNetworkConfig {
+  /**
+   * @param {!Element} autoAmpAdsElement
+   */
+  constructor(autoAmpAdsElement) {
+    this.autoAmpAdsElement_ = autoAmpAdsElement;
+  }
+
+  /**
+   * @param {!Window} unused
+   * @override
+   */
+  isEnabled(unused) {
+    return true;
+  }
+
+  /**
+   *
+   */
+  isResponsiveEnabled() {
+    return true;
+  }
+
+  /** @override */
+  getConfigUrl() {
+    const docInfo = Services.documentInfoForDoc(this.autoAmpAdsElement_);
+
+    const websiteId = this.autoAmpAdsElement_.getAttribute('data-website-id');
```

Comment:
Should we validate this before passing to `buildUrl`?

Reply:
cool, I was thinking you might want to error or something if its not available instead of making the request to your server but totally up to you

Reply:
I wasn't sure if it's be proper to throw an error and I couldn't find any examples in the auto-amp-ads of validation, so I added the check in the `isEnabled` method. It should now return not-enabled when the id is not valid